website/community/committers/release.md

```diff
@@ -425,6 +440,8 @@ If the vote failed, click "Drop" to drop the staging Maven artifacts.
 
 Send the release announcement to `dev@opendal.apache.org` and CC `announce@apache.org`.
 
+> Tips: Please following the [Committer Email](https://infra.apache.org/committer-email.html) guide to make sure you have already set up the email SMTP. Otherwise, your email cannot be sent to the announce mailing list.
```

Comment:
👍

Reply:
Another issue is the mail sent to announce@a.o must be plain text.